api/bases/dataplane.openstack.org_openstackdataplanenodesets.yaml

```diff
@@ -197,6 +197,29 @@ spec:
                 - ctlplaneInterface
                 - deploymentSSHSecret
                 type: object
+              containerImages:
+                properties:
+                  frr:
+                    type: string
+                  iscsiD:
+                    type: string
+                  logrotate:
+                    type: string
+                  neutronMetadataAgent:
+                    type: string
```

Comment:
the neutron sriov nic agent container appears to be missing.

Reply:
Looks like no one ever added it on main. I'll add it there and then rebase this.

Reply:
https://github.com/openstack-k8s-operators/dataplane-operator/pull/760